openedx/core/djangoapps/content_tagging/api.py

```diff
@@ -399,9 +402,12 @@ def tag_object(
             taxonomy=taxonomy,
             tags=tags,
         )
-        CONTENT_OBJECT_TAGS_CHANGED.send_event(
+        CONTENT_OBJECT_ASSOCIATIONS_CHANGED.send_event(
```

Comment:
Do you think we should emit the deprecated event here?

Reply:
Oh oops yes -- we need to emit both events in `content_tagging` while the deprecation process happens. I'll fix this!